notebooks/source/time_series_forecasting.ipynb

```diff
@@ -32,14 +32,14 @@
     "import jax.numpy as np\n",
     "from jax import lax, random, vmap\n",
     "from jax.config import config; config.update(\"jax_platform_name\", \"cpu\")\n",
+    "from jax.nn import softmax\n",
     "\n",
     "import numpyro\n",
     "import numpyro.distributions as dist\n",
     "from numpyro.diagnostics import autocorrelation, hpdi\n",
-    "from numpyro.distributions.util import softmax\n",
     "from numpyro import handlers\n",
     "from numpyro.mcmc import MCMC, NUTS\n",
-    "assert numpyro.__version__.startswith('0.2.0')\n"
+    "assert numpyro.__version__.startswith('0.2.0')"
```

Comment:
I think we'll need this for other examples too?

Reply:
You already added this to all examples/tutorials.